pkg/infra/pulumi_aws/iac/sanitization/sanitizer.ts

```diff
@@ -1,6 +1,9 @@
 import Multimap = require('multimap')
 import * as sha256 from 'simple-sha256'
 
+// Set debug = true to enable additional logging
+let debug = false
```

Comment:
nit: Would it be possible to make this a config value?

Reply:
as in pulumi config?

Reply:
There's also `pulumi.log.[debug|info|etc]`, though I don't know exactly how they work. Maybe the verbose flag sets it?